tests/foreman/cli/test_organization.py

```diff
@@ -1693,3 +1694,32 @@ def test_positive_info_by_name(self):
         org = make_org()
         result = Org.info({'name': org['name']})
         self.assertEqual(org['id'], result['id'])
+
+    @tier1
+    @skip_if_bug_open('bugzilla', 1418412)
+    def test_positive_multibyte_latin1_org_names(self):
+        """Hammer Multibyte and Latin-1 Org names break list pagination
+
+        :id: 4fa0afe7-6d0a-4c3e-a0fc-4ecb95c50fc9
+
+        :BZ: 1418412
+
+        :expectedresults: Multibyte and latin1 names need to be
+            displayed with consistent spacing
+        """
+        org_names = valid_org_names_list()
+        for org in org_names:
+            make_org({'name': org})
+        org_list = Org.list(output_format='table').remove('')
+        for org in org_names:
+            Org.delete({'name': org})
+        if org_list:
+            for org_str in org_list:
+                width = 0
+                for char in org_str:
+                    eaw = unicodedata.east_asian_width(char)
+                    if eaw in ["Na", "N", "A", "H"]:  # Narrow, neutral,...
+                        width += 1
+                    else:  # Wide
+                        width += 2
```

Comment:
Nitpick: Multiple checks and multiple loops made a huge indentation. How about a bit different style to reduce it: for each char add +1 width, and, if it's not in `["Na", "N", "A", "H"]`, add extra +1? Smth like that:
```python
for char in org_str:
    eaw = unicodedata.east_asian_width(char)
    width += 1
    if eaw not in ["Na", "N", "A", "H"]:
        width += 1
```